src/main/java/uk/gov/companieshouse/database/entity/ReasonEntity.java

```diff
@@ -11,15 +11,21 @@
 public class ReasonEntity implements Serializable {
 
     private final OtherReasonEntity other;
+    private final IllnessReasonEntity illnessReason;
 
-    public ReasonEntity(OtherReasonEntity other) {
+    public ReasonEntity(OtherReasonEntity other, IllnessReasonEntity illnessReason) {
         this.other = other;
+        this.illnessReason = illnessReason;
     }
 
     public OtherReasonEntity getOther() {
         return this.other;
     }
 
+    public IllnessReasonEntity getIllnessReason() {
+        return illnessReason;
+    }
```

Comment:
Can these getters be made consistent please?

Reply:
Done